### DISCUSSION TITLE
Can you make similar one for Oxford Dictionary?

### DISCUSSION THREAD

**Discussion by I-am-Zeyad** [johnfactotum/quick-lookup#23] [Ideas]
Wikitionary is not the best. It has some advanced words, and that makes it hard to understand the definition. On the other hand, Oxford Dictionary is a great way to understand the new words.

**Comment by johnfactotum** [comment]
I don't currently have any plans for supporting other dictionary (see [README.md](https://github.com/johnfactotum/quick-lookup#will-you-add-support-for-other-onlineoffline-dictionaries)).

Also note that Oxford Dictionary's API is not free. Well, it does have a free plan but it's very limited (only 1000 requests per month).
